build.sbt

```diff
@@ -39,16 +38,23 @@ lazy val commonSettings = Seq(
     Resolver.sonatypeRepo("snapshots")
   ),
   libraryDependencies ++= Seq(
-    "com.github.mpilquist" %%% "simulacrum" % "0.12.0" % CompileTime,
+    "com.github.mpilquist" %%% "simulacrum" % "0.12.0" % Provided,
     "org.typelevel" %%% "machinist" % "0.6.4",
     compilerPlugin("org.scalamacros" %% "paradise" % "2.1.0" cross CrossVersion.patch),
     compilerPlugin("org.spire-math" %% "kind-projector" % "0.9.6")
   ),
+  pomPostProcess := { (node: xml.Node) =>
+    new RuleTransformer(new RewriteRule {
+      override def transform(node: xml.Node): Seq[xml.Node] = node match {
+        case e: xml.Elem
+          if e.label == "dependency" && e.child.exists(child => child.label == "groupId" && child.text == "com.github.mpilquist") => Nil
+        case _ => Seq(node)
```

Comment:
Is this blanket removing every dependencies under "com.github.mpilquist" for the sake of simulacrum. Is this worthwhile than just using `Provided`?

Reply:
it just removes the simulacrum dependency the POM, a known concern, as done in the past elsewhere- eg https://github.com/typelevel/cats-effect/pull/77#issuecomment-340068924 .

But if you would rather a more fine controlled selection, by artifact name, sure-np

Reply:
yeah, I think artifact name would be better, but if it's trickier than group name I'd vote for just leave it as "Provdided", not ideal, but didn't cause any problems in the past.